pkg/operator/k8sutil/prometheus.go

```diff
@@ -86,6 +87,14 @@ func CreateOrUpdatePrometheusRule(prometheusRule *monitoringv1.PrometheusRule) (
 		if !errors.IsAlreadyExists(err) {
 			return nil, fmt.Errorf("failed to create prometheusRules. %+v", err)
 		}
+		// Get current PrometheusRule so the ResourceVersion can be set as needed
+		// for the object update operation
+		promRule, err := client.MonitoringV1().PrometheusRules(namespace).Get(name, metav1.GetOptions{})
+		if err != nil {
+			return nil, fmt.Errorf("failed to get prometheusRule object. %+v", err)
```

Comment:
```suggestion
			return nil, fmt.Errorf("failed to get prometheusRule object. %v", err)
```

Reply:
@leseb All / most other errors around this added error are using `%+v` should all be changed?

Reply:
Yes, please.